packages/services/src/services/MastheadLogo/MastheadLogo.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+class MastheadLogoAPI {
+  /**
+   * Determines whether to return custom or default IBM logo
+   *
+   * @returns {boolean} Use alternate logo or not
+   *
+   * @example
+   * import { MastheadLogoAPI } from '@carbon/ibmdotcom-services';
+   *
+   * const useAlternateLogo = MastheadLogoAPI.setMastheadLogo(logoData);
+   */
+  static setMastheadLogo(logoData) {
```

Comment:
I'm wondering actually if we should pass in data from an actual data fetch (maybe it lives in IBM.com Translations), so that the app doesn't need to set this manually? Wondering how this can be managed across multiple gateway pages and the Masthead then renders the IBM logo accordingly?

Reply:
@jeffchew I agree. I'm going to remove the prop and the data will optionally come from the translation files.